webapp-django/crashstats/settings/base.py

```diff
@@ -506,10 +506,16 @@ def path(*dirs):
     RAVEN_CONFIG = {
         'dsn': raven_dsn,
         'release': SOCORRO_REVISION,
+        # Defines keys to be sanitized by SanitizeKeysProcessor
+        'sanitize_keys': [
+            'sessionid',
+            'csrftoken',
+            'anoncsrf',
+            'sc',
```

Comment:
What's "sc"?

Reply:
@peterbe I don't know much about the webapp. Can you think of other keys that should be sanitized?

Reply:
I'm not entirely sure. In the context of Google Analytics it means "sessionControl", but here I just saw it set to some hashed or base64'd value and figured that we have no idea what it is and aren't using it currently, so we might as well assume it's sensitive and not collect it until we understand what it is.

Reply:
Yeah, what if you use an API token as a header. Will that be sanitized? E.g. 
`curl -H "Auth-Token: 18af2acef8a54dbca9580e2bb8ba9c9x" https://crash-stats.mozilla.com/api/RawCrash/?uuid=fb27d0f3-db8e-4835-9311-288bb0170829`

<img width="1080" alt="screen shot 2018-02-28 at 8 43 41 am" src="https://user-images.githubusercontent.com/26739/36790654-8f05a412-1c63-11e8-8888-af22883414f4.png">

Perhaps temporarily but in a `raise Exception("Thanks obama!")` in one of the private API endpoints and see if Raven picks up the 'Auth-Token' header. 

Ideally you'd want to programmatically massage it. E.g.
```python
for key in request.headers:
  if key.lower() == 'auth-token':
    request.headers[key] = request.headers[key][:5] + '.' * 10
```
Then you'd be able to see "Yes, this user used an auth-token" and perhaps using your superuser access somehow figure out who that user belongs to `select * from auth_token where token like 'ad4510cd%';`

Reply:
@peterbe I tossed that into my queue of things to look into. I'll write up a bug depending on what I find.

Reply:
Oh, I forgot to mention, I tested this. Auth-Token headers show up under the header list but are marked as `[Filtered]` and you can't see the actual token. Adding it to the list of filtered keys didn't change this behavior so I didn't include it.

Reply:
@Osmose You rock! Thank you!